### PR TITLE
JavaScript code cleanup

### PR DESCRIPTION
**Pull request by thiemowmde**
All pure refactoring with no functional change,

### CODE REVIEW

File: modules/ext.PropertySuggester.EntitySelector.js [hunk 1]
```diff
@@ -16,36 +16,50 @@
...
 		 * @param {string} term
 		 * @return {Object}
 		 */
 		_getSearchApiParameters: function( term ) {
-			var self = this;
-
-			if( !self._useSuggester() ) {
-				return self._oldGetSearchApiParameters( term )
+			if ( !this._useSuggester() ) {
+				return this._oldGetSearchApiParameters( term )
```
Comment (mariushoch) [inline modules/ext.PropertySuggester.EntitySelector.js]: missing semicolon :(

File: modules/ext.PropertySuggester.EntitySelector.js [hunk 2]
```diff
@@ -56,108 +70,120 @@
...
 		_getPropertyContext: function() {
-			if( this._isInNewStatementView() ) {
+			if ( this._isInNewStatementView() ) {
 				return 'item';
-			} else if( this._isQualifier() ) {
+			} else if ( this._isQualifier() ) {
 				return 'qualifier';
-			} else if( this._isReference() ) {
+			} else if ( this._isReference() ) {
 				return 'reference'
```
Comment (mariushoch) [inline modules/ext.PropertySuggester.EntitySelector.js]: missing semicolon

File: modules/ext.PropertySuggester.EntitySelector.js [hunk 3]
```diff
@@ -16,36 +16,50 @@
...
 
 		/**
-		 *
+		 * @inheritdoc
+		 * @protected
 		 * @param {string} term
 		 * @return {Object}
 		 */
 		_getSearchApiParameters: function( term ) {
-			var self = this;
```
Comment (mariushoch) [inline modules/ext.PropertySuggester.EntitySelector.js]: still used below

**Comment by mariushoch** [comment]
I've rebased this and added the missing semicolons in a follow up commit.

**Comment by thiemowmde** [comment]
:+1:

**Pull request merged**